querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/ExtendedBeanSerializer.java

```diff
@@ -30,7 +30,7 @@
 import com.querydsl.sql.codegen.support.PrimaryKeyData;
 
 /**
- * ExtendedBeanSerialzier outputs primary key based equals, hashCode and toString implementations
+ * ExtendedBeanSerializer outputs primary key based equals, hashCode and toString implementations
```

Comment:
Pro tip: if you use `{@link }` comments, IDEs should assist you in (not) making typos.
The same applies for [later on in the diff](https://github.com/querydsl/querydsl/pull/1263/files#diff-bb2c9aa8e1e2b3228ff0918b96f99dd0R35).